05-read-write-plot.Rmd

```diff
@@ -407,36 +428,36 @@ save_tmap(tm  = tmap_obj, filename = "lifeExp_tmap.png")
 <!-- a = plot(world["lifeExp"]) -->
 <!-- ``` -->
 
-On the other hand, an interactive map could be usually saved as an HTML file. 
-This can be done with the `saveWidget()` function from the `htmlwidget` package:
+On the other hand, on can save interactive maps as an HTML file using the `saveWidget()` function from the `htmlwidgets` package:
 
+<!-- example doesn't work, problem with colors I guess -->
```

Comment:
the mapview example doen't work, pls fix

Reply:
That's a bug in mapview. It is already fixed in the dev version on gh, so it should be on CRAN soon.